atd-toolbox/fix_cr3_stored_flag/fix_cr3_stored_flag.py

```diff
@@ -0,0 +1,149 @@
+"""Gets a list of crash ids we have CR3 PDFs for in our AWS S3 bucket and checks them against crashes in the atd_txdot_crashes table where cr3_stored_flag = 'Y'.
+Updates cr3_stored_flag to 'N' if those crash IDs aren't in the list of CR3s we have.
+"""
+
+import argparse
+import requests
+import subprocess
+
+from env import HASURA_AUTH
+
+CRASHES_QUERY = """
+  query QueryCrashes {
+    atd_txdot_crashes(
+      where: { cr3_stored_flag: { _eq: "Y" } }
+    ) {
+      crash_id
+    }
+  }
+"""
+
+CRASHES_UPDATE = """
+  mutation UpdateCrash(
+    $crash_id: Int!,
+  ) {
+    update_atd_txdot_crashes_by_pk(
+        pk_columns: { crash_id: $crash_id },
+        _set: { cr3_stored_flag: "N" }
+    ) {
+        crash_id
+    }
+  }
+"""
+
+
+def get_s3_cr3_files():
+    """Connect to the aws s3 bucket and get all of the names of the cr3 pdfs we have
+    and pipe them into a new file in this directory.
+    """
+    # Creates a new txt file in the current directory to store output from the following commands
+    subprocess.run(["touch", "s3_cr3_file_names.txt"])
+    s3_cr3_file_names = open("s3_cr3_file_names.txt", "w")
+    print(
+        "Writing names of cr3 pdfs we have in s3 to a new file in this directory called 's3_cr3_file_names.txt', this can take a few minutes..."
+    )
+    aws = subprocess.Popen(
+        ["aws", "s3", "ls", "s3://atd-vision-zero-editor/production/cris-cr3-files/"],
```

Comment:
cool to see an alternative to using boto3. Using that library would probably have produced more code too.

Reply:
lol i didnt even think to use that library/didnt know about it!

Reply:
Oh cool - totally valid either way! 😎